internal/code-generator/generator/generator.go

```diff
@@ -32,12 +34,17 @@ func executeTemplate(w io.Writer, name, path, commit string, fmap template.FuncM
 	getCommit := func() string {
 		return commit
 	}
+	stringVal := func(val string) string {
```

Comment:
As this seems to be the definition of this function used in templates, it would be nice to document the purpose of it here.

Reply:
done